globals.go

```diff
@@ -785,6 +785,10 @@ func IsPatchRequester(requester string) bool {
 	return requester == PatchVersionRequester || IsGitHubPatchRequester(requester)
 }
 
+func IsNonPatchRequester(requester string) bool {
+	return utility.StringSliceContains([]string{RepotrackerVersionRequester, TriggerRequester, MergeTestRequester, AdHocRequester}, requester)
```

Comment:
Could we instead just check !IsPatchRequester since it should be one or the other?

Reply:
I think because `!IsPatchRequester` includes git tag requesters SkipOnRequester would skip a git tag requester when the bvt is git tag only.

Reply:
Git tag versions aren't patches though, so it seems incorrect to exclude them from this case, logically, so we may need to either rename to be really explicit (although I don't see when this case would be relevant globally so possibly a private function in model), use the logic directly in the model function, or use the original logic.

Reply:
Got it. This logic is a headache 🤯 .
I went back to the original logic and removed the new stuff.